flink-cdc-common/src/main/java/com/ververica/cdc/common/pipeline/PipelineOptions.java

```diff
@@ -37,7 +37,7 @@ public class PipelineOptions {
     public static final ConfigOption<Integer> PIPELINE_PARALLELISM =
             ConfigOptions.key("parallelism")
                     .intType()
-                    .noDefaultValue()
+                    .defaultValue(1)
```

Comment:
This is a public API change, we only introduce this in 3.1 version, we need some discussion in dev group

Reply:
@leonardBang Thank you for review. I have supplemented the relevant issues accordingly #2855 .